about.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- External styling -->
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <link
+    rel="stylesheet"
+    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+  />
+    <!-- My style -->
+    <link rel="stylesheet" href="about.css">
+    <title>about</title>
+</head>
+<body>
+    <!-- Hamburger Navigation section -->
+    <header>
+        <div class="mobile-nav"> 
+            <i class="fa-solid fa-bars" id="hamburger"></i>
+        </div>
+        <!-- Navigation section -->
+
+        <div class="desktop-nav" id="desktop-nav">
+            <a href="Index.html" id="logo">HackingHerCode</a>
+            <ul id="nav-links" class="close-menu">
+                <li><a href="Index.html" id="home">Home</a></li>
+                <li><a href="about.html">About</a></li>
+                <li><a href="#" id="register">Register</a></li>
+                <li><a href="#" id="sponsors">Sponsors</a></li>
+                <li><a href="#" id="teachers">Teachers</a></li>
+                <li><a href="#" id="last">HackingHerCode</a></li>
+              </ul>
+        </div>
+    </header>
+    <main>
+        <section>
+            <div id="heading">
+                <div>
+                    <!-- About section -->
+                  <h1 id="about-heading">
+                     HackingHercode Bootcamp 2022.
+                  </h1>
+                </div>
+                <div id="about-description">
+                  <p>
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolorum aliquam dolor dicta 
+                    recusandae ad repellat laboriosam facere perferendis 
+                    laudantium iusto libero quis beatae, quo ex. Voluptates debitis eos non sunt?
+                  </p>
+                </div>
+                <div class="about-contact-info">
+                 <p>Please Contact us Via Email: @oyomepriscilla@gmail.com</p> 
+                 <!-- <p>Or at the venue @Priscaville Gardens Badagry</p> -->
+                </div>
+            </div>
+        </section>
+        <section class="about-initiative"> 
+            <!-- About the initiative -->
+            <h2>HackingHerCode Initiative</h2>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                 Nisi, maiores! Amet tenetur ducimus in vero. Quod quasi harum 
+                 totam accusamus laborum magnam ratione minus! Alias, natus. 
+                Culpa facilis sit neque.</p>
```

Comment:
- Please, kindly use the font size of 16px to match the design as much as you can.